app/views/pages/about.html.erb

```diff
@@ -0,0 +1,22 @@
+<div class="container release-notes col-md-12">
+  <h1 class="page-heading">About the New Catalog</h1>
+
+  <p>Initially released on 28 April 2016, this catalog is the result of a nearly two year long project to replace our twenty year old <%= t('blacklight.voyager') %> and incorporate the best features of new types of library search technology.</p>
+
+  <p>The catalog is based on <a href="http://projectblacklight.org/" target="_blank">Blacklight</a>, which is an open source collaboration across several institutions including <a href="https://newcatalog.library.cornell.edu/">Cornell</a>, <a href="https://catalyst.library.jhu.edu/">Johns Hopkins</a>, <a href="https://searchworks.stanford.edu/">Stanford</a> <a href="http://search.library.yale.edu/">Yale</a>, and <a href="http://projectblacklight.org/#examples">a number of others</a>.</p>
+
+  <p>As the examples above demonstrate, Blacklight is highly configurable and customizable, which provides a number of advantages:</p>
+
+  <ul>
+    <li><strong>Tighter Integration with Existing Services.</strong> We do not have to rely on vendors or wait for support for intergration with the several different requesting and fulfillment options that the library has to offer.</li>
```

Comment:
There is an extra 'r' in 'integration'.  'intergration'.

We do not have to rely on vendors or wait for support for integration with the several different requesting and fulfillment options the library has to offer.